### ISSUE TITLE
Android 14 bug: One of RECEIVER_EXPORTED or RECEIVER_NOT_EXPORTED should be specified

### ISSUE THREAD

**Issue by RobinBobin** [dooboolab-community/react-native-iap#2718]
`com.android.billingclient:billing-ktx:6.0.1` makes apps using `react-native-iap` crash when run on Android 14:

```
One of RECEIVER_EXPORTED or RECEIVER_NOT_EXPORTED should be specified when a receiver isn't being registered exclusively for system broadcasts
```

com.android.billingclient:billing-ktx:6.1.0 is safe for use.

**Environment:**

- react-native-iap: "12.10.5" / "12.13.0"
- react-native: "0.73.4"
- Platforms (iOS, Android, emulator, simulator, device): Android

**Comment by PavelVisme** [comment]
Thank you very much. You are very smart person, your fix helped me

**Comment by ukcasso** [comment]
The resolved version has been uploaded to 12.13.1. Thank you.

https://github.com/dooboolab-community/react-native-iap/pull/2717
https://github.com/dooboolab-community/react-native-iap/releases/tag/12.13.1
